webapp/channels/src/components/purchase_modal/renewal_card.scss

```diff
@@ -0,0 +1,258 @@
+.PurchaseModal {
+    .RenewalRHS {
+        position: sticky;
+        display: flex;
+        width: 25%;
+        flex-direction: column;
+        padding-top: 80px;
+
+        .RenewalCard {
+            max-width: 353px;
+            padding: 28px 32px;
+            border: 1px solid var(--light-8-center-channel-text, rgba(61, 60, 64, 0.08));
+            background: var(--light-center-channel-bg, #fff);
+            border-radius: var(--spacing-xxxxs, 4px);
```

Comment:
does that css variable name actually exist?

Reply:
I don't think so. Figma spits these out. Removed.